contracts/contracts/auction/TokenDepositLocker.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.8;
+
+import "./DepositLocker.sol";
+import "../token/TrustlinesNetworkToken.sol";
+
+/*
+  The ETHDepositLocker contract locks ETH deposits
+
+  For more information see DepositLocker.sol
+*/
+
+contract TokenDepositLocker is DepositLocker {
+    TrustlinesNetworkToken public token;
```

Comment:
It should be better to use a token interface to keep it generic as it was the mentality we had on these contracts